api/v3/Case/Getdetailscount.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Case.getdetailscount API
+ *
+ * Provides a count of cases but properly respects filters unlike `getcount`.
+ * Please refer to C51-277 for more information.
+ *
+ * @param array $params
+ * @return array API result
+ * @throws API_Exception
+ */
+function civicrm_api3_case_getdetailscount($params) {
+  $params['options'] = CRM_Utils_Array::value('options', $params, []);
+  $params['options']['is_count'] = 1;
+
+  $casesList = civicrm_api3('Case', 'getdetails', $params);
+
+  return $casesList['values'];
```

Comment:
In this case will `$casesList['values']` be an number, i.e. the count?

Reply:
That is correct. Should the `@return` doc change?

Reply:
Nope, I just wanted to check. I know normally `values` is an array of entities, but I guess then with `is_count = 1` it becomes a number.